superset/translations/ko/LC_MESSAGES/messages.po

```diff
@@ -917,4027 +917,3125 @@ msgstr ""
 msgid "Select charts"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2089
-#: superset/assets/src/explore/controls.jsx:2107
-#: superset/assets/src/explore/controls.jsx:2138
 msgid "Error while fetching charts"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2101
 msgid "Right Axis chart(s)"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2104
 msgid "Choose one or more charts for right axis"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2118
 msgid "Prefix metric name with slice name"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2125
 msgid "Reverse Lat & Long"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2132
 msgid "deck.gl charts"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2135
 msgid "Pick a set of deck.gl charts to layer on top of one another"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2148
 msgid "Javascript data interceptor"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2149
 msgid ""
-"Define a javascript function that receives the data array used in the "
-"visualization and is expected to return a modified version of that array. This "
-"can be used to alter properties of the data, filter, or enrich the array."
+"Define a javascript function that "
+"receives the data array used in "
+"the visualization and is expected to "
+"return a modified version of that "
+"array. This can be used to alter"
+" properties of the data, filter, or"
+" enrich the array."
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2155
 msgid "Javascript data mutator"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2156
 msgid ""
-"Define a function that receives intercepts the data objects and can mutate it"
+"Define a function that receives "
+"intercepts the data objects and can "
+"mutate it"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2160
 msgid "Javascript tooltip generator"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2161
 msgid ""
-"Define a function that receives the input and outputs the content for a tooltip"
+"Define a function that receives the "
+"input and outputs the content for "
+"a tooltip"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2165
 msgid "Javascript onClick href"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2166
 msgid "Define a function that returns a URL to navigate to when user clicks"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2171
 msgid "Extra data for JS"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2173
 msgid "List of extra columns made available in Javascript functions"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2178
 msgid "Stroked"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2180
 msgid "Whether to display the stroke"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2186
 msgid "Filled"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2188
 msgid "Whether to 
```

Comment:
Ideally these translations (security) should be included on flask-appbuilder

Reply:
https://github.com/dpgaspar/Flask-AppBuilder/tree/master/flask_appbuilder/translations

You mean this repository??

Reply:
yes

Reply:
Thanks for answer

I will raise pull request soon